cogs/assistance-cmds/aroma.wiiu.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Aroma
 url: https://wiiu.hacks.guide/#/aroma/getting-started
+aliases: getstarted
```

Comment:
Does this actually work?

I'm not sure if the backend code is able to handle aliases being the same as an already-existing command.

I'm actually not sure if the reverse works either, it might cause `aroma 3ds` to be a valid command (unless this was fixed?)

imo we should deprecate `aroma` and just move directly to `getstarted wiiu`

Reply:
if the wii u folks are fine with that i'm game

Reply:
im fine with that.